.travis.yml

```diff
@@ -16,6 +16,10 @@ before_install:
 script:
   - ./.ci/build.sh
   - rm -rf private/
+  - >
```

Comment:
nit:
On second thought, not a bad idea to add [this comment](https://github.com/coala/coala/blob/master/.travis.yml#L76) here as well